src/workbook/stakeholder-sheet/holdings-columns.ts

```diff
@@ -38,6 +39,8 @@ export class StakeholderColumn {
     myColumn
       .createNestedRange()
       .addBlankCell(Styles.default)
+      .addCell("Options Remaining for Issuance")
+      .addBlankCell(Styles.default)
```

Comment:
**Non-blocking:** If you put the `{style: Styles.default}` as a parameter to the nested range, you shouldn't need to pass it on LL41-43